packaging/opae/rpm/create

```diff
@@ -66,56 +83,29 @@ do
     mkdir -p "${RPMBUILD_DIR}/$d"
 done
 
+# clone opae legancy and tests
+
+cd "${SDK_DIR}/.."
+rm -rf _build
+mkdir _build
+cd _build
+cmake .. -DOPAE_BUILD_LEGACY=ON  -DOPAE_BUILD_TESTS=ON
+
 # Create source tarball.
 cd "${SDK_DIR}/.."
-tar --transform='s/opae-sdk/opae/' \
-  --exclude=.git \
-  --exclude=.gitignore \
-  --exclude=.github \
-  --exclude=.travis.yml \
-  --exclude=opae.spec.in \
-  --exclude=opae.spec \
-  --exclude=opae-libs/external \
-  --exclude=opae-libs/tests \
-  --exclude=opae-libs/plugins/ase \
-  --exclude=opae-libs/cmake/config/libopae-all.spec.in \
-  --exclude=opae-libs/cmake/config/run_coverage_test.sh.in \
-  --exclude=opae-libs/cmake/config/run_coverage_test_local.sh.in\
-  --exclude=external \
-  --exclude=platforms \
-  --exclude=samples/base \
-  --exclude=samples/hello_afu \
-  --exclude=samples/hello_mpf_afu \
-  --exclude=samples/intg_xeon_nlb \
-  --exclude=samples/base \
-  --exclude=scripts \
-  --exclude=tests \
-  --exclude=tools/fpgametrics \
-  --exclude=tools/libboard/board_dc \
-  --exclude=tools/extra/ras \
-  --exclude=tools/extra/fpgabist \
-  --exclude=tools/extra/pac_hssi_config \
-  --exclude=tools/extra/fpgadiag \
-  --exclude=tools/extra/c++utils \
-  --exclude=tools/extra/pyfpgadiag \
-  --exclude=tools/extra/pypackager \
-  --exclude=tools/utilities \
-  --exclude=opae-libs/include/opae/cxx/.clang-format \
-  --exclude=opae-libs/libopaecxx/.clang-format \
-  --exclude=opae-libs/pyopae/.clang-format \
-  --exclude=opae-libs/.clang-format \
-  --exclude=opae-libs/.clang-format \
-  -z -c -f opae.tar.gz opae-sdk
-
-mv opae.tar.gz "${RPMBUILD_DIR}/SOURCES"
+
+${SDK_DIR}/scripts/create-tarball.sh opae-${version}
```

Comment:
Why call out to a separate script for this?

Reply:
create-tarball.sh can be used in different scripts in feature.

Reply:
The idea behind the build structure in the /packaging directory is that it would contain everything needed to make the build happen.  ie, to build the opae RPM, everything needed should be located in packaging/opae/rpm.  You're assuming that both the RPM .spec file and this external script will be in other locations.  We should move them into packaging/opae/rpm.